iosApp/Survey/Sources/Presentation/ViewId/ViewId+Account.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewId+Account.swift
+//  Survey
+//
+//  Created by Bliss on 27/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+extension ViewId {
+
+    enum Account: String {
+
+        case view = "account.view"
```

Comment:
@blyscuit Optional, using `accountView` is better IMO 🙏

Reply:
I think the idea here is `view` is equivalent to `self`. `view` is under the enum `Account` or any other screens so `Account.view`/`Login.view`/etc. is already describing `AccountView`. What do you think?

Reply:
That's good, we can keep it 👌